src/traits.rs

```diff
@@ -0,0 +1,34 @@
+#[cfg(feature = "sys")]
+mod sys {
+    use echo::*;
+    pub trait TypeInfo {
```

Comment:
Is this still needed?

Reply:
Oops, my bad. I've forgot to remove the file, sorry.